Gems/ScriptCanvas/Code/Editor/Assets/ScriptCanvasFileHandling.cpp

```diff
@@ -169,10 +170,9 @@ namespace ScriptCanvasEditor
             return AZ::Failure(AZStd::string("Failed to find graph data after loading source"));
         }
 
-        return AZ::Success();
+        return AZ::Success(deserializationErrors);
```

Comment:
it is quite weird to wrap error inside success object, maybe having a reference parameter like what you did for LoadObjectFromStringByType?

Reply:
See nodes above. Serialization errors to imply object construction failure.